imxvpuapi/imxvpuapi.h

```diff
@@ -400,7 +400,7 @@ typedef struct
 {
 	/* Stride of the Y and of the Cb&Cr components.
 	 * Specified in bytes. */
-	unsigned int y_stride, cbcr_stride;
+	unsigned int y_stride, cbcr_stride, total_size;
```

Comment:
Hm. Can you provide some example pseudo code which shows a case where this is inconvenient? Is it because the generalized allocator would have to somehow keep the ImxVpuFramebufferSizes around?